app/views/landing_page/_hero.erb

```diff
@@ -41,10 +43,13 @@
                   </g>
                 </svg>
               </div>
-              <input type="search" name="q" class="hero__search-input" placeholder="<%= s["search_placeholder"]["value"] %>"/>
+              <input type="search" name="q" class="hero__search-input<%= combined_mode %>" placeholder="<%= s["search_placeholder"]["value"] %>"/>
+              <% if search_mode == "keyword_and_location_search" %>
+                <input type="search" name="lq" class="hero__search-input<%= combined_mode %>" placeholder="<%= s["search_location_with_keyword_placeholder"]["value"] %>"/>
```

Comment:
We should change this so that the code is backwards compatible, i.e. doesn't crash for old JSON structure without `search_location_with_keyword_placeholder` key in the structure.

So,

```erb
<%= s["search_location_with_keyword_placeholder"]["value"] %>
```

...should be changed to:

```erb
<%= s.dig("search_location_with_keyword_placeholder", "value") %>
```